### ISSUE TITLE
Error receiving the JSONP result

### ISSUE THREAD

**Issue by odiumediae** [revolunet/react-mailchimp-subscribe#7]
I'm probably just as thick as a tar slushly here, and at this time I'm almost scared to ask, but I have no choice.

I'm developing a React app for a client and I've got two forms on a landing page. Just the email and the submit button. I'm using `create-react-app` and I've updated the start script in order to start the app with SSL by default: `HTTPS=true react-scripts start`.

Everything works flawlessly, apart from the Mailchimp form. I can embed it alright, style it, etc. – no problem, but when I send the form data I'm getting an error in Firefox:

> Loading failed for the <script> with source “https://[account name].us17.list-manage.com/subscribe/post-json?u=[u id here]&amp;id=[id here]&EMAIL=[url encoded email here]&c=__jp0”.

I don't know what this error is supposed to tell me exactly, but when I enter the URL in a browser I'm getting a pain text JSONP string back, so I guess the error message means that the browser can't deal with the result from Mailchimp.

The result that I get when I enter the URL in the browser, however, shows that requests are being processed by Mailchimp:

> 
__jp0({"result":"success","msg":"Thank you for subscribing!"})

I'm at my wits end, so I'm grateful for any help here. What am I missing or doing wrong here?

**Comment by revolunet** [comment]
no idea, in the network panel, what is the request status ? 404? 500?

**Comment by odiumediae** [comment]
That's the weird thing, neither a 404 nor a 500, since I'm getting the response as mentioned above, but the browser didn't accept it. After trying out different potential solutions I'm pretty sure the problem are my tracking options in commbination with a possible bug in Firefox, becaue the latter didn't tell me that the response hasn't been accepted, because of the tracking options. After I switched the tracking options a few times, though, Firefox finally gave me an error that said my tracking option settings didn't allow to accept the response (JSONP string). Now it works.

I don't know what caused this, but it's definitely not a problem with the library, so I'll close this issue.

Sorry, if I've wasted your time. I surely wasted my own ;)
